apps/meteor/tests/e2e/page-objects/omnichannel-departments.ts

```diff
@@ -63,4 +63,20 @@ export class OmnichannelDepartments {
 	get btnModalConfirmDelete() {
 		return this.page.locator('#modal-root .rcx-modal .rcx-modal__footer .rcx-button--danger');
 	}
+
+	get upgradeDepartmentsModal() {
+		return this.page.locator('[data-qa-id="enterprise-departments-modal"]');
+	}
+
+	get btnUpgradeDepartmentsModalClose() {
+		return this.page.locator('#modal-root .rcx-modal .rcx-modal__header .rcx-button--small-square');
```

Comment:
<img src="https://media2.giphy.com/media/3OhXBaoR1tVPW/giphy.gif"/>

Using CSS selectors as locator is not recommended. We don't have a better way to locate the close button elsewhere?